plugin/pxr/maya/lib/usdMaya/usdTranslatorExport.mel

```diff
@@ -153,7 +153,11 @@ global proc int usdTranslatorExport (string $parent,
         checkBox -l "Export UVs" exportUVsCheckBox;
 
         checkBox -l "Export ColorSets" exportColorSetsCheckBox;
-        checkBox -l "Export Reference Objects" exportReferenceObjectsCheckBox;
+
```

Comment:
That's the Pixar plugin. Perhaps you meant to make these changes in `plugin\adsk\scripts\mayaUsdTranslatorExport.mel` instead?

Reply:
No, we don't really need the option in the UI at this stage. Just missed file name and saw the same option name declared.

Reply:
I made the update because I got the impression that the option being changed in the shared translators seems to be referenced by the Pixar plugin as well. Is that correct?